parsers/RE.py

```diff
@@ -0,0 +1,140 @@
+    3: {'coal': 1, 'biomass': 0},
+    4: {'coal': 1, 'biomass': 0},
+    5: {'coal': 1, 'biomass': 0},
+    6: {'coal': 1, 'biomass': 0},
+    7: {'coal': 0.61, 'biomass': 0.23},
+    8: {'coal': 0.61, 'biomass': 0.4},
+    9: {'coal': 0.61, 'biomass': 0.4},
+    10: {'coal': 0.61, 'biomass': 0.4},
+    11: {'coal': 0.61, 'biomass': 0.4},
+    12: {'coal': 1, 'biomass': 0.23},
+}
+
+
+def fetch_production(zone_key='RE', session=None, target_datetime=None,
+                     logger=logging.getLogger(__name__)):
+
+    if target_datetime:
+        raise NotImplementedError('There is no historical data')
+    else:
+        to = arrow.now(tz='Indian/Reunion')
+
+    r = session or requests.session()
+
+    params = {
+        'dataset': 'prod-electricite-temps-reel',
+        'timezone': 'Indian/Reunion',
+        'sort': 'date',
+        'rows': 288
+    }
+    response = r.get(API_ENDPOINT, params=params)
+    data = json.loads(response.content)
+    datapoints = []
+
+    for i in range(0, len(data['records'])):
+        record = data['records'][i]
+        # All biomass
+        biomass = 0
+        if(MAP_GENERATION['bagasse_coal'] in record['fields']):
+            biomass += MAP_GENERATION_BAGASSE_COAL[to.month]['biomass'] * record['fields'][MAP_GENERATION['bagasse_coal']]
+        if(MAP_GENERATION['biomass'] in record['fields']):
+            biomass += record['fields'][MAP_GENERATION['biomass']]
+
+        wind = 0
+        solar = 0
+        gas = 0
+        coal = 0
+
+        # Other
+        if(MAP_GENERATION['wind'] in record['fields']):
+            wind = record['fields'][MAP_GENERATION['wind']]
+        if(MAP_GENERATION['solar'] in record['fields']):
+            solar = record['fields'][MAP_GENERATION['solar']]
+        if(MAP_GENERATION['gas'] in record['fields']):
+            gas = record['fields'][MAP_GENERATION['gas']]
+        if(MAP_GENERATION['oil'] in record['fields']):
+            oil = record['fields'][MAP_GENERATION['oil']]
+        if(MAP_GENERATION['bagasse_coal'] in record['fields']):
+            coal = MAP_GENERATION_BAGASSE_COAL[to.month]['coal'] * record['fields'][MAP_GENERATION['bagasse_coal']]
+
+        # Storages
+        solar_storage = 0
+        hydro_storage = 0
+        hydro = 0
+        if(MAP_GENERATION['solar_with_storage'] in record['fields']):
+            if(record['fields'][MAP_GENERATION['solar_with_storage']] < 0):
+                solar_storage = record['fields'][MAP_GENERATION['solar_with_storage']]
+            else:
+                solar += record['fields'][MAP_GENERATION['solar_with_storage']]
+        if(MAP_GENERATION['hydro'] in record['fields']):
+            if(record['fields'][MAP_GENERATION['hydro']] < 0):
+                hydro_storage = record['fields'][MAP_GENERATION['hydro']]
+            else:
+                hydro = record['fields'][MAP_GENERATION['hydro']]
+
+        datapoints.append({
+            'zoneKey': 'RE',
+            'datetime': record['fields']['date'],
```

Comment:
I checked the parser output and the `datetime` field needs to be a python `datetime` object.
Right now it's a string. You basically need to parse this string value into a python `datetime` object and then we'll be good to go!

Reply:
Oh thank you, I missed it. I'll do it soon :)